charts/fluid/fluid/crds/data.fluid.io_vineyardruntimes.yaml

```diff
@@ -9,16 +9,61 @@ metadata:
 spec:
   group: data.fluid.io
   names:
+    categories:
+    - fluid
     kind: VineyardRuntime
     listKind: VineyardRuntimeList
     plural: vineyardruntimes
+    shortNames:
+    - vineyard
```

Comment:
How about using v6d for shortname?

Reply:
Just kidding. You can decide by yourself.

Reply:
> How about using v6d for shortname?

Make sense.